packages/@react-spectrum/textfield/src/TextArea.tsx

```diff
@@ -42,17 +46,17 @@ function TextArea(props: SpectrumTextFieldProps, ref: RefObject<TextFieldRef>) {
     if (inputRef.current) {
```

Comment:
Should this be a `useLayoutEffect` so we update immediately instead of waiting for paint?